okhttp/build.gradle

```diff
@@ -22,23 +22,23 @@ task copyJavaTemplates(type: Copy) {
 }
 
 dependencies {
-  api deps.okio
-  api deps.kotlinStdlib
-  compileOnly deps.conscrypt
-  compileOnly deps.android
-  compileOnly deps.jsr305
-  compileOnly deps.animalSniffer
+  api 'com.squareup.okio:okio'
+  api "org.jetbrains.kotlin:kotlin-stdlib:${versions.kotlin}"
```

Comment:
why versions here?

Reply:
I kept this one specific version here because I figured you might want to keep the stdlib in sync with the gradle plugin so you only have one place to bump versions:

```
    classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:${versions.kotlin}"
```

If this isn't an important constraint then we could absolutely just treat the kotling stdlib like all other dependencies.